test/browser/features/translation.feature

```diff
@@ -0,0 +1,20 @@
+@mock-api:address-success @success
+Feature: Happy Path - changing languages
+  Change languages based on cookie
+
+  Background:
+    Given Authenticalable Address Amy is using the system
```

Comment:
Maybe `Authenticable` or `Authenticatable`

Reply:
I can raise this in a separate PR as that would require a change to all 13 test.feature files